### PR TITLE
chore: fix init_node script(grpc-web port)

### PR DESCRIPTION
**Pull request by dudong2**
## Description

fix init_node.sh for grpc-web port(added by #468)
closes: #XXXX

## Checklist:

- [ ] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Review by zemyblue — APPROVED** [review]
LGTM

File: init_node.sh [hunk 1]
```diff
@@ -94,6 +94,7 @@ RPC_PORT=26657
 P2P_PORT=26656
 PROF_PORT=6060
 GRPC_PORT=9090
+GRPC_WEB_PORT=9091
```
Comment (zemyblue) [inline init_node.sh:97]: Is `9091` official port number?

Comment (dudong2) [reply init_node.sh:97]: Yes, default grpc-web port number in cosmos-sdk is 9091.

**Review by 0Tech — APPROVED** [review]
LGTM

**Pull request merged**